docs/production-deployment/cloud/service-health.mdx

```diff
@@ -11,41 +11,38 @@ tags:
   - health
 ---
 
-Temporal Cloud metrics help you to monitor the Service health of your production deployments.
+Temporal Cloud metrics help you to monitor the Service health of your production deployments. This documentation addresses some common best practices for monitoring Service health.
```

Comment:
```suggestion
Temporal Cloud metrics help monitor the service health of production deployments.
This documentation covers best practices for monitoring service health.
```

I really want to know if we're going to stick with Docs single line sentences. I've found it helpful, and it makes localizing changes and diffs far simpler but Jwahir is not an adherent and I'm the only team member of the old regime, and I put an "abolish" item into the next (non-retro) meeting.

Reply:
I am genuinely just very bad at remembering to adhere to this convention and I've never used it elsewhere, so I manage to forget it every time I task switch off of this repo :(

I'm not necessarily a fan, but I also don't hate it nearly as much as I must appear to.